tests/publiccloud/migration.pm

```diff
@@ -21,42 +21,117 @@ use utils;
 use publiccloud::utils;
 
 our $target_version = get_required_var('TARGET_VERSION');
-our $not_clean_vm = get_var('PUBLIC_CLOUD_NO_CLEANUP_ON_FAILURE');
+our $not_clean_vm   = get_var('PUBLIC_CLOUD_NO_CLEANUP_ON_FAILURE');
 
 sub run {
     my ($self, $args) = @_;
     select_serial_terminal();
     my $provider = $args->{my_provider};
-    my $instance = $provider->create_instance();
-    $instance->wait_for_guestregister();
+    my $instance = $provider->create_instance(check_guestregister => 0);
```

Comment:
this change is already in master , please rebase to latest master

Reply:
will do